drawing/tools/custom/Vector.js

```diff
@@ -162,8 +162,8 @@ var Vector = oo.declare(
 				};
 			}
 			var radius = this.util.length(obj);
-			var angle = positioning.angle(obj.start, obj);
-			angle<0 ? angle = 360 + angle : angle;
+			var angle = this.util.angle(obj);
+			angle = angle<0 ? 360 + angle : angle;
```

Comment:
Perhaps to avoid this type of mistake again, we should make it `angle = (angle<0) ?`